containers-tests/tests/map-properties.hs

```diff
@@ -1,5 +1,7 @@
 {-# LANGUAGE CPP #-}
 {-# LANGUAGE TupleSections #-}
+{-# LANGUAGE TypeOperators #-}
+{-# LANGUAGE TypeFamilies #-}
```

Comment:
As brought up in #1165, we would like to get tests working with MHS, which does not support type families. So this is potentially setting up a future problem.